### PR TITLE
Add land value to household inputs

### LINKED ISSUE #161: Land value tax

### LINKED ISSUE #214: Allow households to enter land value
On `Your household` page, for simulating LVT. Would be good for the LVT presentation/paper.

### LINKED ISSUE #215: Set MTR line chart to not interpolated
Extremely minor but we can use this: https://plotly.com/python/line-charts/#interpolation-with-line-plots

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #215 
Fixes #215

### CODE REVIEW

File: policy_engine/app.py [hunk 1]
```diff
@@ -26,11 +26,10 @@
     mtr_chart,
     budget_chart,
 )
-
 from openfisca_uk_data import FRS_WAS_Imputation
 
 VERSION = "0.0.11"
-USE_CACHE = True
+USE_CACHE = False
```
Comment (MaxGhenis) [inline policy_engine/app.py:32]: should this be kept as False?

File: policy_engine/situations/charts.py [hunk 2]
```diff
@@ -27,9 +27,11 @@ def budget_chart(baseline: IndividualSim, reformed: IndividualSim) -> str:
     """
     df = pd.DataFrame(
         {
-            "Employment income": baseline.calc("employment_income").sum(),
-            "Baseline": baseline.calc("net_income").sum(),
-            "Reform": reformed.calc("net_income").sum(),
+            "Employment income": baseline.calc("employment_income").sum(
```
Comment (MaxGhenis) [inline policy_engine/situations/charts.py:30]: I think I removed these `axis=0`s earlier as they seem extraneous per the default. Are they not? Or if you want to be explicit let's just be consistent, as I don't think it's everywhere.

**Review by MaxGhenis — APPROVED** [review]
Going to merge to play around with it, we can check on these questions in a future PR.

File: policy_engine/app.py (hunk 1 above)
Comment (nikhilwoodruff) [reply policy_engine/app.py:32]: Oh no that's my mistake

File: policy_engine/situations/charts.py (hunk 2 above)
Comment (nikhilwoodruff) [reply policy_engine/situations/charts.py:30]: Yeah this actually necessary (caused a bug when removed), [axis=0 is not the same as axis=None](https://numpy.org/doc/stable/reference/generated/numpy.sum.html). This calc() function returns an array of shape (number of people in the household, number of earnings scenarios). Summing by axis=0 here sums to the household level for each earnings scenario. Summing by axis=None (which is essentially axis=(0, 1)) transforms to a scalar, the total net income by all household members in all earnings scenarios.

**Pull request merged**